src/frontend/src/pages/HomePage/WorkPackagesByTimelineStatus.tsx

```diff
@@ -61,24 +94,41 @@ const WorkPackagesByTimelineStatus: React.FC = () => {
             <Card
               variant="outlined"
               key={wbsPipe(wp.wbsNum)}
-              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default }}
+              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default, borderRadius: '16px' }}
             >
-              <CardContent sx={{ padding: 3 }}>
-                <Link
-                  variant="h6"
-                  component={RouterLink}
-                  to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
-                  sx={{ marginBottom: 2 }}
-                >
-                  {wbsPipe(wp.wbsNum)} - {wp.name}
-                </Link>
-                <DetailDisplay label="End Date" content={datePipe(wp.endDate)} paddingRight={2} />
-                <DetailDisplay label="Progress" content={percentPipe(wp.progress)} paddingRight={2} />
-                <DetailDisplay label="Engineering Lead" content={fullNamePipe(wp.projectLead)} paddingRight={2} />
-                <DetailDisplay label="Project Manager" content={fullNamePipe(wp.projectManager)} paddingRight={2} />
-                <Typography>
+              <CardContent sx={{ padding: 2 }}>
+                <Grid container spacing={2}>
+                  <Grid item xs={9}>
+                    <Typography variant="h6" sx={{ textDecoration: 'underline', align: 'right' }}>
+                      {wp.wbsNum.projectNumber} - {wp.projectName}
```

Comment:
we want this to be a link too right, also isnt this just the project number, dont we want the project wbs number, you can probably make a pipe for it that takes in a wbs number and just returns the car.proj.0 of any wbs num